packages/storybook-ui-components/package.json

```diff
@@ -18,8 +18,8 @@
   },
   "homepage": "https://github.com/cockroachdb/ui#readme",
   "dependencies": {
-    "@cockroachlabs/icons": "^0.7.6",
-    "@cockroachlabs/ui-components": "^0.7.7",
+    "@cockroachlabs/icons": "workspace:../icons",
+    "@cockroachlabs/ui-components": "workspace:../ui-components",
```

Comment:
hooray `workspace:` protocol: https://pnpm.io/workspaces#workspace-protocol-workspace 🎉

Reply:
It shouuuuuuuld automatically link everything together :)